packages/docs/pages/form.tsx

```diff
@@ -239,7 +239,7 @@ const FormPage = () => {
                       const [value, setValue] = useState('BigCommerce');
                       const [error, setError] = useState(ERROR_MSG);
 
-                      const handleSubmit = (event) => {
+                      const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
🍹 We generally like to use the component props to type `event`. In case we ever change the `onSubmit` api, we'll be able to tell more easily what's wrong.
```suggestion
                      const handleSubmit: FormProps['onSubmit'] = (event) => {
```